mypy/build.py

```diff
@@ -1466,6 +1498,27 @@ def process_graph(graph: Graph, manager: BuildManager) -> None:
             process_stale_scc(graph, scc)
 
 
+def order_ascc(graph: Graph, ascc: AbstractSet[str], pri_max: int = PRI_ALL) -> List[str]:
+    """Come up with the ideal processing order within an SCC."""
```

Comment:
Explain what this algorithm does in more detail. Describe the time complexity. It would be nice to have unit tests for this, as the code looks a little tricky.